### PR TITLE
[New Release] ml-auto-eda release 1.1

### PR DESCRIPTION
**Pull request by luotigerlsx**
The new release adds the following capabilities:

- Ensure Python3 runtime;
- Consistent numeric notation;
- Append job configuration to the final report;
- Performance improvement through parallel query execution;
- Generate different formats of report: markdown, html, pdf;
- Export analysis result as dictionary;
- Sample dataset for statistical test;

There is also a major code structure refactoring to improve readability.

Internal review has been done by other sces as well.

### CODE REVIEW

**Comment by luotigerlsx** [comment]
@jaketf Can you help take a look at the PR. Thanks.

**Comment by jaketf** [comment]
@luotigerlsx FYI @TheMichaelHu is our new reviewer for assets in the ML domain.

File: tools/ml-auto-eda/README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
 # Auto Data Exploration and Feature Recommendation Tool
 
-Machine learning (ML) projects typically start with a comprehensive exploration of the provided datasets.
-It is critical that ML practitioners to gain a deep understanding of:
-- the properties of the data.
-- the quality of the data.
+Machine learning (ML) projects typically start with a comprehensive exploration of the provided datasets.
```
Comment (TheMichaelHu) [inline tools/ml-auto-eda/README.md:3]: fyi: whitespace added, not sure if intentional

**Comment by TheMichaelHu** [comment]
@luotigerlsx Can you rebase onto master?

**Comment by luotigerlsx** [comment]
> @luotigerlsx Can you rebase onto master?

@TheMichaelHu I am not able to do the rebase actually. Can you help, thanks :)

**Comment by TheMichaelHu** [comment]
Just waiting for #532 to fix cloud build.

**Pull request merged**